src/Applications/LDAS_App/ldas_setup

```diff
@@ -18,7 +18,7 @@ from dateutil.relativedelta import relativedelta
 
 
 """
-This script is intended to be run from ../Application/LDAS_App or ../Linux/bin
+This script is intended to be run from ../Application/LDAS_App or ../install/bin
```

Comment:
Please delete the `../Applications/LDAS_App` from this line and all other lines please.

Reply:
done 1586a5c8cd5514f95afc4bfa4368a4cb4ca5ef62